### PR TITLE
Gruwella class 8

### PR DESCRIPTION
**Pull request by gruwella**
Not ready to merge yet!  So far I have slides for Arrays and Strings.  I still need to add in the last few concepts for the module.  Please let me know if you have any suggestions for making this better!

### CODE REVIEW

File: modules/module_05/code/Multiples.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+//Objective: Print out the multiples of 7
+//Concepts: Initializing and indexing into arrays
+public class Multiples {
```
Comment (andrewjensen) [inline modules/module_05/code/Multiples.java]: I thought the intent of this example was a little ambiguous, so I would recommend using a simpler example at that point in the lecture, maybe with just Strings and no computation.

File: modules/module_05/homework.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
+# Module 05 Homework
```
Comment (andrewjensen) [inline modules/module_05/homework.md]: Thanks for merging this into the PR. I'll make sure to split the homework code out into separate files.

File: modules/module_05/slides.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+```
+String[] animals = {"tiger", "kangaroo", "shark"};
+System.out.println(animals[0]); // tiger
+System.out.println(animals[2]); // shark
+System.out.println(animals.length); // 3
+```
+
+-----------------------------------------------------------------------------
+# Practice!
+- Open up Multiples.java
```
Comment (andrewjensen) [inline modules/module_05/slides.md]: See my above comment in `Multiples.java`

**Comment by gruwella** [comment]
I changed that Multiples.java example to a more simple one like you recommended.  It's probably better to stay simple anyway since we might be short on time.  I think the slides are pretty much done now, unless you have any more suggestions to improve them.  Speaking of which, how should we divide up the time teaching?

**Comment by andrewjensen** [comment]
We spoke on the phone. We think the slides are ready to go and we need just a few more quiz questions. I'll merge the homework branch into this one, then squash and merge everything into master.

**Pull request merged**